### ISSUE TITLE
Upgrade to use jitsi-meet stable-8252

### ISSUE THREAD

**Issue by kpeiruza** [jitsi-contrib/jitsi-helm#71]
Hi, I've tested changing the applicationVersion at Chart.yaml and prosody/Chart.yaml + increasing chart version to 1.3.3 and it works like a charm, but I see you get docs/index.yaml and a tarball also in this repo.

If I want to contribute such a PR, will docs/* get regenerated automatically or I need to update that index and create the clean tarball on my own?

Best regards,

**Comment by spijet** [comment]
Hello @kpeiruza!

Usually I package the tarball and rebuild the repo index manually on every release. I'll review the changes in the new version and update the chart soon, thank you for the notification.

Meanwhile, you can update your installation to new images even when using the existing chart release. To do that, you can use `<foo>.image.tag` values, like this:
```yaml
web:
  image:
    tag: "stable-8252"
jvb:
  image:
    tag: "stable-8252"
jibri:
  image:
    tag: "stable-8252"
jicofo:
  image:
    tag: "stable-8252"
prosody:
  image:
    tag: "stable-8252"
```

**Comment by spijet** [comment]
Fixed in 8510c72.
